modules/swagger-codegen/src/test/resources/2_0/petstore.yaml

```diff
@@ -701,6 +701,15 @@ definitions:
         type: string
       message:
         type: string
+  PetStatus:
```

Comment:
@JFCote I'll remove this tomorrow add the enum example to https://github.com/swagger-api/swagger-codegen/blob/master/modules/swagger-codegen/src/test/resources/2_0/petstore-with-fake-endpoints-models-for-testing.yaml instead.

When the Play generator, we should consider using petstore-with-fake-endpoints-models-for-testing.yaml, which covers a lot more edge cases.